### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by jontje**
As per title

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ RobotWare versions less than `6.07.01` are now incompatible with *abb_libegm* (d
 
 ## Overview
 
-A C++ library for interfacing with ABB robot controllers supporting *Externally Guided Motion* (EGM). See the *Application manual - Controller software IRC5* for a detailed description of what EGM is and how to use it.
+A C++ library for interfacing with ABB robot controllers supporting *Externally Guided Motion* (EGM). See the *Application manual - Externally Guided Motion* for a detailed description of what EGM is and how to use it.
```
Comment (gavanderhoorn) [inline README.md]: Does this have a document number? If so, I suggest we add it (including the version/revision).

File: README.md [hunk 2]
```diff
@@ -4,10 +4,15 @@
 [![license - bsd 3 clause](https://img.shields.io/:license-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
 [![support level: vendor](https://img.shields.io/badge/support%20level-vendor-brightgreen.png)](http://rosindustrial.org/news/2016/10/7/better-supporting-a-growing-ros-industrial-software-platform)
 
-## Important Note
+## Important Notes
 
 RobotWare versions less than `6.07.01` are now incompatible with *abb_libegm* (due to changes in the EGM communication protocol).
 
+Pull request https://github.com/ros-industrial/abb_libegm/pull/63 turned this package from a Catkin package, into a plain CMake package. ROS users can utilize any of the following build tools to build the source code:
```
Comment (gavanderhoorn) [inline README.md]:
```suggestion
Pull request [ros-industrial/abb_libegm#63](https://github.com/ros-industrial/abb_libegm/pull/63) turned this package from a Catkin package into a plain CMake package. ROS users may use any of the following build tools to build the library:
```

File: README.md [hunk 3]
```diff
@@ -4,16 +4,23 @@
 [![license - bsd 3 clause](https://img.shields.io/:license-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
 [![support level: vendor](https://img.shields.io/badge/support%20level-vendor-brightgreen.png)](http://rosindustrial.org/news/2016/10/7/better-supporting-a-growing-ros-industrial-software-platform)
 
-## Important Note
+## Important Notes
 
 RobotWare versions less than `6.07.01` are now incompatible with *abb_libegm* (due to changes in the EGM communication protocol).
 
+Pull request [#63](https://github.com/ros-industrial/abb_libegm/pull/63) turned this package from a Catkin package, into a plain CMake package. ROS users can utilize any of the following build tools to build the source code:
```
Comment (gavanderhoorn) [inline README.md]:
```suggestion
Pull request [abb_libegm#63](https://github.com/ros-industrial/abb_libegm/pull/63) turned this package from a Catkin package, into a plain CMake package. ROS users can utilize any of the following build tools to build the source code:
```

File: README.md (hunk 1 above)
Comment (jontje) [reply README.md]: Yes, it has, and I will add it.

File: README.md [hunk 4]
```diff
@@ -4,16 +4,23 @@
...
 
-## Important Note
+## Important Notes
 
 RobotWare versions less than `6.07.01` are now incompatible with *abb_libegm* (due to changes in the EGM communication protocol).
 
+Pull request [abb_libegm#63](https://github.com/ros-industrial/abb_libegm/pull/63) turned this package from a Catkin package, into a plain CMake package. ROS users can utilize any of the following build tools to build the source code:
+
+* ROS 1: `catkin_make_isolated` or [catkin_tools](https://catkin-tools.readthedocs.io/en/latest/index.html).
+* ROS 2: `colcon`.
```
Comment (gavanderhoorn) [inline README.md]:
```suggestion
* ROS 2: [colcon](https://colcon.readthedocs.io/en/released/).
```

File: README.md (hunk 2 above)
Comment (gavanderhoorn) [reply README.md]: This actually also contains some updates to the rest of the sentence. Somehow I'd 'fixed' the way the PR link is displayed twice.

Comment (jontje) [reply README.md]: I saw them and they should already have been added.

File: README.md [hunk 5]
```diff
@@ -4,16 +4,23 @@
...
 
+Pull request [abb_libegm#63](https://github.com/ros-industrial/abb_libegm/pull/63) turned this package from a Catkin package into a plain CMake package. ROS users may use any of the following build tools to build the library:
+
+* ROS 1: `catkin_make_isolated` or [catkin_tools](https://catkin-tools.readthedocs.io/en/latest/index.html).
+* ROS 2: [colcon](https://colcon.readthedocs.io/en/released/).
+
 ## Overview
 
-A C++ library for interfacing with ABB robot controllers supporting *Externally Guided Motion* (EGM). See the *Application manual - Controller software IRC5* for a detailed description of what EGM is and how to use it.
+A C++ library for interfacing with ABB robot controllers supporting *Externally Guided Motion* (EGM). See the *Application manual - Externally Guided Motion* (document ID: `3HAC073319-001`, revision: `B`) for a detailed description of what EGM is and how to use it.
```
Comment (gavanderhoorn) [inline README.md]: Is this document publicly available? I cannot find anything with that document nr.

Comment (jontje) [reply README.md]: Hm... I can find it in the `ABB Library`, but it is tagged as `PREMIUM` so I don't think it is publicly available. I usually access the `ABB Library` via RobotStudio:

Capture

Comment (gavanderhoorn) [reply README.md]: As long as users of this library would have access to it.

Just wanted to make sure.

Comment (jontje) [reply README.md]: I can't imagine that users wouldn’t have access to the documentation if they have purchased the EGM license.

**Comment by gavanderhoorn** [comment]
Thanks @jontje :+1:

**Pull request merged**